h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -97,14 +99,25 @@ export default function AnnotationCard({
               <AnnotationTimestamps
                 annotationCreated={annotation.created}
                 annotationUpdated={annotation.updated}
+                withEditedTimestamp={annotation.updated !== annotation.created}
               />
             </div>
-            {group && (
-              <div className="flex gap-x-1 items-baseline flex-wrap-reverse">
-                <AnnotationGroupInfo group={group} />
-                <AnnotationDocument annotation={annotation} />
-              </div>
-            )}
+            <div className="flex gap-x-1 items-baseline flex-wrap-reverse">
+              {group && (
+                <>
+                  <AnnotationGroupInfo group={group} />
+                  <AnnotationDocument annotation={annotation} />
+                </>
+              )}
+              {isReply && (
+                <div
+                  className="ml-auto flex items-center gap-x-1"
+                  data-testid="reply-indicator"
+                >
+                  <ReplyIcon /> This is a reply
+                </div>
```

Comment:
There are no designs on how this should look like, so I decided to add a simple indicator in a location where it's easy to spot, but I'm open to suggestions and ideas.

I also considered making this a link that scrolls you to the annotation that is referenced, but considering that annotation may or may not have been loaded yet, that could lead to inconsistent behaviors, so I decided to delay that for now.

Reply:
Nothing in particular. You suggested the "Reply. See [thread]" format and for some reason I thought you were implying we should remove the icon as well, so I was just confirming.

I'll keep it. All good.